.gitignore

```diff
@@ -97,7 +97,6 @@ CMakeCache.txt
 /cmake-build-debug
 /cmake_build
 /build
-/cmake_build*
```

Comment:
Why is this PR messing with `.gitignore`?

Reply:
Common problem I find myself caught in. Pro tip: https://stackoverflow.com/a/1753078 ... I've started adding my "personal" Git ignore patterns to `.git/info/exclude` instead.